package.json

```
@@ -9,7 +9,8 @@
     ".": {
       "import": "./dist/index.mjs",
       "require": "./dist/index.js"
-    }
+    },
+    "./": "./"
```

this is the most backward-compatible way to do this

Alternatively, if we want very specific control:
```suggestion
    "./package.json": "./package.json"
```